src/Mvc/Mvc.Abstractions/src/ModelBinding/ModelMetadata.cs

```diff
@@ -433,11 +437,11 @@ internal IReadOnlyList<ModelMetadata> BoundProperties
     /// Gets a value indicating whether <see cref="ModelType"/> is a complex type.
     /// </summary>
     /// <remarks>
-    /// A complex type is defined as a <see cref="Type"/> without a <see cref="TypeConverter"/> that can convert
-    /// from <see cref="string"/>. Most POCO and <see cref="IEnumerable"/> types are therefore complex. Most, if
+    /// A complex type is defined as a <see cref="Type"/> that is NOT <see cref="IsConvertibleType"/> and NOT <see cref="IsParseableType"/>.
```

Comment:
You shouldn't `<see cref="IsConvertibleType"` since the property isn't public. Can we keep the previous doc about `TypeConverters` and include details on how what constitutes a parseable type?

Reply:
Oh! that is true. Let me fix it.